resources/decision_tree.json

```diff
@@ -12,12 +12,103 @@
         ]
```

Comment:
`test_lb` and `test_lp` should use a tree without VKGL and Clinvar. separate test tree based on default tree?